src/style/_layout.scss

```diff
@@ -56,7 +56,10 @@
         background: rgba($opblock-tag-background-color-hover,.02);
     }
 
-    @include text_headline();
+    h4
+    {
+      @include text_title($size: $opblock-tag-h4-font-size);
```

Comment:
Just mentioning here - I think it's probably within the scope of this PR to set default font-size for text elements based on UX doc and swaggerhub styleguide. That being said - the `$size` param here is still necessary for overwrites